app/views/hosts/rebuild_config.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render 'selected_hosts', :hosts => @hosts %>
+
+<%= form_tag submit_rebuild_config_hosts_path(:host_ids => params[:host_ids]), :onsubmit => "resetSelection()" do %>
+  <small><%= _('Rebuild configuration for selected hosts') %></small>
```

Comment:
It'd be worth saying "Rebuild orchestration configuration" here to match the API, so it's clearer what it's for.  There's limited space on the dropdown menu to say precisely what config it is, but here's better.